inc/forms/class-wp-forms-form-handler.php

```
@@ -1,658 +1,347 @@
+            return;
+        }
+
+        add_action('wpforms_process_complete', [$this, 'handle_form_submission'], 10, 4);
+        add_action('wpforms_form_settings_notifications', [$this, 'add_form_settings']);
+        add_filter('wpforms_builder_settings_sections', [$this, 'add_settings_section']);
+    }
+
+    /**
+     * Handle form submission and trigger Bento event
+     */
+    public function handle_form_submission($fields, $entry, $form_data, $entry_id) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$entry'. (Unused Code Rules)

(UnusedFormalParameter)